examples/kubernetes/sc.yaml

```diff
@@ -4,4 +4,8 @@ metadata:
   name: glusterfs-csi
   annotations:
     storageclass.beta.kubernetes.io/is-default-class: "true"
+parameters:
+  restURL: "http://192.168.121.17:24007"
+  restUser: "glustercli"
+  restSecret: "79e8744fb454cce883dc395ecb9455296895591bd19dfdbcc50dbd2a90012ad4"
```

Comment:
may be we have to reconsider the parameter names once again, however for now change `resturl`, restuser, restsecret` to `gd2url,`gd2user` and `gd2userpassword`.

Reply:
I disagree. Naming these fields after implementation specifics (gd2) makes the SC specification less user friendly. Additionally, camel case w/ lower first letter is pretty standard for kube fields.

I would suggest: glusterURL, clusterURL, serverURL, or even glusterCluster.
This will eventually be a service address that redirects to an arbitrary Gluster pod, so glusterCluster is probably the most accurate description. The fact that it's a URL is rather obvious from the format.